### PR TITLE
21504 row group header tooltip

### PR DESCRIPTION
**Pull request by fellmann**
## Summary

- [Ticket](https://flipapp.myjetbrains.com/youtrack/agiles/115-84/current?issue=APP-21504)
- 
### General

- [x] [Changeset added](https://github.com/changesets/changesets/blob/main/docs/intro-to-using-changesets.md)
- [x] The submitted code is organized and formatted according to our [💅 Style Guide](https://swirl-storybook.flip-app.dev/?path=/docs/requirements-style-guide--docs).
  - [x] The code is formatted with Prettier.
  - [x] There are no linting errors.

### CODE REVIEW

File: packages/swirl-components/src/components/swirl-table-row-group/swirl-table-row-group.spec.tsx [hunk 1]
```diff
@@ -23,4 +23,30 @@ describe("swirl-table-row-group", () => {
       </swirl-table-row-group>
     `);
   });
+
+  it("renders its label and content", async () => {
```
Comment (Sqrrl) [inline packages/swirl-components/src/components/swirl-table-row-group/swirl-table-row-group.spec.tsx:27]: Nitpick: "renders with tooltip"

**Comment by Sqrrl** [comment]
Small nitpick. Looks good otherwise.

File: packages/swirl-components/src/components/swirl-table-row-group/swirl-table-row-group.spec.tsx [hunk 2]
```diff
@@ -23,4 +23,30 @@ describe("swirl-table-row-group", () => {
...
+    });
+
+    expect(page.root).toEqualHtml(`
+      <swirl-table-row-group class="table-row-group" label="Label" role="rowgroup" tooltip="Test">
+        <mock:shadow-root>
+          <div class="table-row-group__header-row" role="row">
+            <span aria-rowspan="0" class="table-row-group__label" role="rowheader">
+              Label
+              <span class="table-row-group__tooltip">
+                <swirl-tooltip content="Test" position="right">
```
Comment (Sqrrl) [inline packages/swirl-components/src/components/swirl-table-row-group/swirl-table-row-group.spec.tsx:40]: Quesion: Is "top" the better default position? The component will automatically flip sides if there isn't enough space.

Comment (fellmann) [reply packages/swirl-components/src/components/swirl-table-row-group/swirl-table-row-group.spec.tsx]: yes, matches the design, too

**Pull request merged**